### PR TITLE
Added user sign up

### PR DESCRIPTION
**Pull request by kenwmak**

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (droberts84) [inline spec/models/user_spec.rb]: Do you have tests you want to create for the `User` model?  Either write them or delete this file

**Comment by droberts84** [comment]
It looks like you may have customized some of the Devise views (or considered it)?  Could you tell me a bit a about the changes you made, if any?

BTW, its helpful to put a high level summary in the PR of the changes contained

**Comment by droberts84** [comment]
:+1:

**Comment by kenwmak** [comment]
Sure, I added name as one of the field in the sign up page.

**Pull request merged**